packages/php-wasm/universal/src/lib/base-php.ts

```diff
@@ -174,6 +175,60 @@ export abstract class BasePHP implements IsomorphicLocalPHP, Disposable {
 			throw new Error('Invalid PHP runtime id.');
 		}
 		this[__private__dont__use] = runtime;
+		this[__private__dont__use].ccall(
+			'wasm_set_phpini_path',
+			null,
+			['string'],
+			[PHP_INI_PATH]
+		);
+
+		if (!this.fileExists(PHP_INI_PATH)) {
+			this.writeFile(
```

Comment:
Why don't we use `setPhpIniEntries` here?

Reply:
Because PHP_INI_PATH does not exist yet and `setPhpIniEntries` would fail. We could perhaps change `setPhpIniEntries` semantics to "create the file if it's missing", but this feels off – I want BasePHP to guarantee the file will be there later on and get a clear error if it isn't found at a later stage.

Reply:
And if we ever want to do type-checking of known PHP ini settings, we could just write the blank file, and use `setPhpIniEntries` to write those initial settings.